contrib/subtree/git-subtree.sh

```diff
@@ -392,6 +392,26 @@ find_existing_splits () {
 	done
 }
 
+find_mainline_ref () {
+	debug "Looking for first split..."
+	dir="$1"
+	revs="$2"
+	main=
+	sub=
```

Comment:
What are `main` and `sub`?

Reply:
An artifact of starting with a copy of find_existing splits that should be removed. Since you’re building on top of this, is it better to rewrite the history or fix in a new commit?

Reply:
Rewriting history won't bother me, my WIP is going to need to be edited and rebased significantly anyway.